downstream/modules/platform/proc-restore-aap-backup.adoc

```diff
@@ -0,0 +1,21 @@
+[id="proc-restore-aap-backup_{context}"]
+
+= Restore backup to new instance
+
+Running the `./setup.sh` script with the `restore_backup_file` flag will migrate content from the backup file of your original 1.x instance to the new instance, effectively migrating all job histories, templates, and other {PlatformNameShort} related content.
```

Comment:
Suggest rewording: Running the `./setup.sh` script with the `restore_backup_file` flag migrates content from the backup file of your original 1. instance to the new instance. This effectively migrates all job histories, templates, and other {PlatformNameShort} related content.